src/painter.rs

```diff
@@ -78,22 +79,13 @@ impl Painter {
         prompt_offset: (u16, u16),
         cursor_index_in_buffer: usize,
     ) -> Result<()> {
-        let new_index = cursor_index_in_buffer;
-        let offset = original_line.len() - new_index;
-        // Repaint logic:
-        //
-        // Start after the prompt
-        // Paint the buffer
-        // Move back to the column based on the offset
-        // Finally, reset the cursor to the saved position
```

Comment:
Is this description not up to date anymore?

Reply:
I think that now it's not useful to describe the process anymore, it's auto explanatory.

Reply:
OK